src/status_im/common/alert_banner/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im.common.alert-banner.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.alert-banner.style :as style]
+            [utils.re-frame :as rf]))
+
+(def colors-map
+  {:alert
+   {:background-color colors/warning-50-opa-20
+    :text-color       colors/warning-60}
+   :error
+   {:background-color colors/danger-50-opa-20
+    :text-color       colors/danger-60}})
+
+(defn- banner
+  [{:keys [text type]} second-banner?]
+  [rn/view (when second-banner? {:style style/second-banner-wrapper})
+   [hole-view/hole-view
+    {:style (style/hole-view (get-in colors-map [type :background-color]))
+     :holes (if second-banner?
+              []
+              [{:x            0
+                :y            40
+                :width        (:width (rn/get-window))
+                :height       40
+                :borderRadius 20}])}
+    [quo/text
+     {:size   :paragraph-2
+      :weight :medium
+      :style  {:color (get-in colors-map [type :text-color])}}
+     text]]])
+
+(defn view
+  []
+  (let [banners       (rf/sub [:alert-banners])
+        banners-count (count banners)
+        alert-banner  (:alert banners)
+        error-banner  (:error banners)
+        safe-area-top (safe-area/get-top)]
+    [hole-view/hole-view
+     {:style {:background-color colors/neutral-100}
+      :holes [{:x            0
+               :y            (+ safe-area-top (* 40 banners-count))
+               :width        (:width (rn/get-window))
+               :height       40
+               :borderRadius 20}]}
+     [rn/view
+      {:style {:height           safe-area-top
+               :background-color (get-in colors-map
+                                         [(if error-banner :error :alert) :background-color])}}]
+     (when error-banner
+       [banner error-banner false])
+     (when alert-banner
+       [banner alert-banner (= banners-count 2)])]))
```

Comment:
can there ever be more than 2 banners?

Reply:
Hopefully not, otherwise we will only have banners on screen :smile: . I don't think there will be more than 2, currently figma only has two types :alert and :error. But if we will have more in future, we can modify accordingly.

Reply:
sounds good 👍